docs/eks-installation.md

```diff
@@ -0,0 +1,30 @@
+# Deploying Antrea in AWS EKS
+
+This document describes steps to deploy Antrea in NetworkPolicy only mode to an AWS EKS cluster.
+
+Assuming you already have an EKS cluster, and have KUBECONFIG environment variable pointing to
+the kubeconfig file of that cluster.
+
+To deploy the latest version of Antrea (built from the master branch) to EKS, get the Antrea EKS
+deployment yaml at:
+```
+https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-eks.yml
```

Comment:
The link does not work for me. Should be: https://github.com/vmware-tanzu/antrea/blob/master/docs/ipsec-tunnel.md?

Reply:
we just need to check in build/yamls/build/antrea-eks.yml, right? Do I miss anything? I suspect link does not work because it is not checked in yet. I use the same link u have inhttps://github.com/vmware-tanzu/antrea/blob/master/docs/ipsec-tunnel.md for IPSec i.e

https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-ipsec.yml

Reply:
Ok. My mistake. Sorry.